bluemira/equilibria/coils/_grouping.py

```diff
@@ -171,8 +171,42 @@ def fix_sizes(self):
         """
         Fix the sizes of coils in CoilGroup
         """
-        for fx in self.__list_getter("fix_size"):
-            fx()
+        self.__run_func("fix_size")
```

Comment:
What's the advantage `__run_func` over just calling the function? It's not clear to me what `__run_func` is doing. I think it could do with a bit more explanation on why it's needed/what it offers that directly calling the function(s) does not.

Reply:
In this case its just to avoid repeat code, but in more complex situations you may be passing weirdly shaped object in a nested way into a nested coil groups function eg a circuit within a group being the simplest (complex) situation